### ISSUE TITLE
Add Existing Components to API Gateway

### ISSUE THREAD

**Issue by giobart** [oakestra/oakestra#70]
Dispatch requests as follows:

/api -> system manager
/api/net -> network manager 

expose the api gateway to port 20000

**Comment by michaelschlicker** [comment]
Yes I am working on it as part of the introduction of a multi cloud broker.

I chose nginx over kong because currently and for the foreseeable future only basic features are needed which don't require a configuration via a web interface.
In order to keep the overhead small I've decided to go with the more lightweight nginx.

The API gateway will be used to support multiple cloud provider plugins via a common multi cloud route which can then call individual cloud providers.
Once the API gateway is available over time more and more parts of the system can be switched from addressing via ports to using the gateway's routing feature.
